External/TestDataSet/rawdata/sub-Sub1/perf/sub-Sub1_m0scan.json

```diff
@@ -0,0 +1,48 @@
+{
+	"Modality":"MR",
+	"MagneticFieldStrength":3,
+	"ImagingFrequency":127.744,
+	"Manufacturer":"Philips",
+	"ManufacturersModelName":"Ingenuity",
+	"SoftwareVersions":"3.2.2_3.2.2.1",
+	"MRAcquisitionType":"2D",
+	"ScanningSequence":"GR",
+	"SequenceVariant":"SK",
+	"ScanOptions":"FS",
+	"AcquisitionTime":[14,17],
+	"AcquisitionVoxelSize":[3.4,3.4,4.5],
+	"PhilipsRWVSlope":1.25543,
+	"PhilipsRWVIntercept":0,
+	"PhilipsRescaleSlope":1.25543,
+	"PhilipsRescaleIntercept":0,
+	"PhilipsScaleSlope":0.000267712,
+	"UsePhilipsFloatNotDisplayScaling":1,
+	"SliceThickness":4.5,
+	"SpacingBetweenSlices":4.5,
+	"SAR":0.0145759,
+	"EchoTime":0.010487,
+	"RepetitionTimePreparation":10,
+	"FlipAngle":90,
+	"CoilString":"SENSE-Head-8",
+	"PartialFourier":0.435484,
+	"PercentPhaseFOV":100,
+	"EchoTrainLength":27,
+	"PhaseEncodingSteps":62,
+	"AcquisitionMatrixPE":62,
+	"ReconMatrixPE":64,
+	"PixelBandwidth":3054,
+	"PhaseEncodingAxis":"i",
+	"ImageOrientationPatientDICOM":[0.988042,-0.143706,0.0558729,0.154183,0.918592,-0.363889],
+	"InPlanePhaseEncodingDirectionDICOM":"COL",
+	"ConversionSoftware":"dcm2niix",
+	"ConversionSoftwareVersion":"v1.0.20181125  (JP2:OpenJPEG) (JP-LS:CharLS) GCC5.5.0",
+	"M0":"separate_scan",
+	"readout_dim":"2D",
+	"Vendor":"Philips",
+	"bReproTesting":0,
+	"Q":{
```

Comment:
remove Q and this time also the content.

Reply:
58c05c6